README.md

```diff
@@ -103,6 +103,15 @@ switch(x) {
 - C and C++ support a set of
   [digraph and trigraph](https://en.wikipedia.org/wiki/Digraphs_and_trigraphs#C) tokens to
   accommodate certain archaic keyboards. Trigraphs were removed from C++ in C++17.
+- ISO C forbids conversion of a function pointer to an object pointer type:
+  ```cpp
+  void (*func_ptr)() = dlsym(mylib, "func"); // gcc yields a warning with standard C17 in pedantic mode
+  ```
+  However, if taking the address to the function pointer first, then casting to `void**` and finally dereferencing this pointer again, makes it work without warnings:
```

Comment:
I think there's room for improvement on this sentence, maybe it could be reworded something like this?
```suggestion
  Casting the pointer from `dlsym` is forbidden but a trick to do this without warnings is to do the conversion on the left hand side as `&func_ptr` is an object pointer:
```